source/welcome.svelte

```diff
@@ -0,0 +1,126 @@
+<svelte:options customElement='rgh-welcome' />
```

Comment:
This file is a bit messy. Suggestions on how to make it more "svelte-like" very much welcome.